src/InfoBoxes/Panel/RadioEdit.cpp

```diff
@@ -30,8 +30,7 @@ Copyright_License {
 
 class RadioOffsetButtons final : public OffsetButtonsWidget {
 public:
-  RadioOffsetButtons(bool active_freq) noexcept
-    :OffsetButtonsWidget(UIGlobals::GetDialogLook().button, _T("%.0f KHz"), 5, 1000),set_active_freq(active_freq) {}
+  RadioOffsetButtons(bool active_freq):OffsetButtonsWidget(UIGlobals::GetDialogLook().button, _T("%.0f kHz"), 5, 1000),set_active_freq(active_freq) {}
```

Comment:
* Readd the noexcept
* Readd the linebreak

Make your commit message more meaningful:
InfoBox/RadioEdit: fix typo in frequency unit

Reply:
Thank you for contributing!